www/include/reporting/dashboard/template/viewHostGroupLog.ihtml

```diff
@@ -155,20 +155,20 @@
         maxDate: '-1',
         onSelect : function(data){  
             jQuery( "select[name='period'] > option[value='']" ).prop('selected', true);
-            jQuery( "input:[value='custom'][name='period_choice'][type='radio']").prop('checked', true);
+            jQuery( "input[value='custom'][name='period_choice'][type='radio']").prop('checked', true);
         }
     });
 
     jQuery("#EndDate").datepicker({
         maxDate: '-1',
         onSelect : function(data){  
             jQuery( "select[name='period'] > option[value='']" ).prop('selected', true);
-            jQuery( "input:[value='custom'][name='period_choice'][type='radio']").prop('checked', true);
+            jQuery( "input[value='custom'][name='period_choice'][type='radio']").prop('checked', true);
```

Comment:
remove withspace after the first parenthisis

Reply:
I can't, as this line was removed in the next commit